modules/books.js

```diff
@@ -0,0 +1,45 @@
+export default class Book {
+  constructor() {
+    this.bookStore = JSON.parse(localStorage.getItem('singleBook')) || [];
+  }
+
+  getBookList() {
```

Comment:
- [ ] In keeping with the ES6 syntax requirement, kindly refactor this function to be an arrow function, like so:

  ```
  getBookList = () => {
    ...
  ```